provider/aws/builds.go

```diff
@@ -339,6 +339,7 @@ func (p *AWSProvider) BuildImport(app string, r io.Reader) (*structs.Build, erro
 				log.Error(err)
 				return nil, err
 			}
+			targetBuild.Id = sourceBuild.Id
```

Comment:
I believe this line would be set around 410-411 with the other `targetBuild` properties.